Tests/LibWeb/Text/expected/WebAudio/AudioBuffer.txt

```diff
@@ -0,0 +1,18 @@
+Error creating AudioBuffer: 'NotSupportedError: Number of channels must not be '0''
```

Comment:
nit: IMO tests generally belong in the same commit that adds the thing being tested